.github/workflows/publish-prisma-schema-wasm.yml

```diff
@@ -42,7 +42,7 @@ jobs:
 
       - uses: actions/setup-node@v3
         with:
-          node-version: "14.x"
+          node-version: '14.x'
```

Comment:
Unrelated to this PR, but shouldn't we use '16.x' here?

Reply:
16 or up would be better indeed, though here it doesn't matter which version we use as long as it works
The upgrade is done in https://github.com/prisma/prisma-engines/pull/4370